README.md

```diff
@@ -27,9 +27,11 @@ alexa_client = AlexaVoiceServiceClient(
 )
 alexa_client.connect()  # authenticate and other handshaking steps
 with open('./tests/resources/alexa_what_time_is_it.wav', 'rb') as f:
-    alexa_response_audio = alexa_client.send_audio_file(f)
-with open('./output.wav', 'wb') as f:
-    f.write(alexa_response_audio)
+    alexa_response_audio_list = alexa_client.send_audio_file(f)
+
+for alexa_response_audio in alexa_response_audio_list:
+    with open('./output.' + alexa_response_audio_list.index(alexa_response_audio) + '.wav', 'wb') as f:
```

Comment:
this would be more pythonic
```
for counter, alexa_response_audio in enumerate(alexa_response_audio_list):
    file_name = './output.{counter}.wav'.format(counter=counter)
    with open(file_name, 'wb') as f:
```
but I'm wondering if a list is the best form of return value. We could instread use `yield` and make `parse_response` a generator